lib/Wikia/src/Service/User/Permissions/PermissionsConfigurationImpl.php

```diff
@@ -418,10 +418,12 @@ private function loadGroupsChangeableByGroups() {
 		$this->groupsAddableByGroup['content-reviewer'] = array( 'content-reviewer' );
 		$this->groupsRemovableByGroup['content-reviewer'] = array( 'content-reviewer' );
 
-		$this->groupsAddableByGroup['vstf'] = array( 'rollback', 'bot' );
-		$this->groupsRemovableByGroup['vstf'] = array( 'rollback', 'bot' );
-		$this->groupsSelfAddableByGroup['vstf'] = array( 'sysop' );
-		$this->groupsSelfRemovableByGroup['vstf'] = array( 'sysop', 'bureaucrat' );
+		$this->groupsSelfAddableByGroup['vstf'] = array( 'rollback', 'bot', 'sysop' );
+		$this->groupsSelfRemovableByGroup['vstf'] = array( 'rollback', 'bot', 'sysop', 'bureaucrat' );
```

Comment:
Minor: Our coding conventions are to [prefer PHP's short array syntax](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax).